### PR TITLE
Fix ResourceTable lookups for pre-expansion and code validation.

### PR DESCRIPTION
**Pull request by dmuylwyk**

### CODE REVIEW

**Comment by dmuylwyk** [comment]
@jamesagnew, please review. Thanks!

File: hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseHapiTerminologySvcImpl.java [hunk 1]
```diff
@@ -956,7 +957,8 @@ private void expandWithoutHibernateSearch(IValueSetConceptAccumulator theValueSe
 
 	@Override
 	public boolean isValueSetPreExpandedForCodeValidation(ValueSet theValueSet) {
-		Long valueSetResourcePid = IDao.RESOURCE_PID.get(theValueSet);
+		ResourceTable resourceTable = (ResourceTable) myValueSetResourceDao.readEntity(theValueSet.getIdElement(), null);
+		Long valueSetResourcePid = resourceTable.getId();
```
Comment (jamesagnew) [inline hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseHapiTerminologySvcImpl.java]: These two lines could probably be refactored into a utility method

Comment (dmuylwyk) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseHapiTerminologySvcImpl.java]: Thanks. Will make this change before I merge.

Comment (dmuylwyk) [reply hapi-fhir-jpaserver-base/src/main/java/ca/uhn/fhir/jpa/term/BaseHapiTerminologySvcImpl.java]: Done.

**Pull request merged**